EIPS/eip-FOCIL.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Committee-based, Fork-choice enforced Inclusion Lists (FOCIL)
+description: Allow a committee of validators to force-include a set of transactions in every block
+author: Thomas Thiery <thomas.thiery@ethereum.org>, Francesco D'Amato <francesco.damato@ethereum.org>, Julian Ma <julian.ma@ethereum.org>, Barnabé Monnot <barnabe.monnot@ethereum.org>, Terence Tsao <ttsao@offchainlabs.com>, Jacob Kaufmann <jacob.kaufmann@ethereum.org>, Jihoon Song <jihoonsong.dev@gmail.com>
+
+discussions-to: <URL>
```

Comment:
Please create a discussions topic in Eth Magicians using the template: https://ethereum-magicians.org/c/eips/5